src/briefcase/integrations/android_sdk.py

```diff
@@ -1069,11 +1068,16 @@ def start_emulator(self, avd):
 
                         # Try again in 2 seconds...
                         self.sleep(2)
-        except Exception:
-            # if the emulator exited, this should return its output immediately;
-            # if it is still running, this will quickly time out and print nothing.
-            with suppress(subprocess.TimeoutExpired):
-                self.tools.logger.info(emulator_popen.communicate(timeout=1)[0])
+        except BaseException:
+            # if the emulator exited or the user sent CTRL+C, show the
+            # available output from the emulator to the user before exiting.
+            output_deadline = time.time() + 1
```

Comment:
Given a lot more is being done now....this seems like a potentially low deadline...especially on slow machines.